.travis.yml

```diff
@@ -11,22 +11,21 @@ matrix:
   allow_failures:
     - os: windows
 before_install:
-- go get -u github.com/client9/misspell/cmd/misspell
-- go get -u golang.org/x/lint/golint
-- go get -u honnef.co/go/tools/cmd/staticcheck
-- go get github.com/fzipp/gocyclo
-- go get golang.org/x/tools/cmd/cover
-# Install gcc, from https://travis-ci.community/t/go-cant-find-gcc-with-go1-11-1-on-windows/293/5
-- if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then choco install -y mingw; export PATH=/c/tools/mingw64/bin:"$PATH"; fi
+  - GO111MODULE=off go get -u github.com/client9/misspell/cmd/misspell
+  - GO111MODULE=off go get -u golang.org/x/lint/golint
+  - GO111MODULE=off go get github.com/fzipp/gocyclo
+  - GO111MODULE=off go get -u honnef.co/go/tools/cmd/staticcheck
+ # - GO111MODULE=off go get golang.org/x/tools/cmd/cover
 before_script:
-- GOFILES=$(find . -type f -name '*.go' | grep -v vendor)
+  - GOFILES=$(find . -type f -name '*.go' | grep -v vendor | grep -v client)
 script:
 # Just check gofmt on linux, it's the fastest builder
 - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then test -z $(gofmt -s -l $GOFILES); fi
-- go test ./... -race -coverprofile=coverage.txt -covermode=atomic
+# - go test ./... -race -coverprofile=coverage.txt -covermode=atomic
 - misspell -error -locale US $GOFILES
 - gocyclo -over 50 $GOFILES
 - golint -set_exit_status $GOFILES
-- staticcheck ./...
+- staticcheck ./cmd/*/*.go ./pkg/*/*.go *.go
```

Comment:
There's no `./pkg/*/*.go` directory in this project.

Reply:
That is correct we do not have a ./pkg/*/*.go directory in this project yet